utt/privacy-wallet-service/proto/api/v1/wallet-api.proto

```diff
@@ -101,6 +101,23 @@ message GetStateResponse {
     map<string, uint64> coins = 4;
 }
 
+message SetAppDataRequest {
+    repeated string keys = 1;
```

Comment:
minor comment: keep kv as single message proto and make that repeated to avoid misuse. You could also consider map<string, string>